### PR TITLE
HS-108 add property bedrooms

### PR DESCRIPTION
**Pull request by suesitran**

### CODE REVIEW

File: lib/features/add_property/view/widgets/add_rooms_view.dart [hunk 1]
```diff
@@ -0,0 +1,135 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:hatspace/features/add_property/view_model/add_property_cubit.dart';
+import 'package:hatspace/strings/l10n.dart';
+
+import 'package:hatspace/gen/assets.gen.dart';
+import 'package:hatspace/theme/widgets/hs_buttons.dart';
+
+class AddRoomsView extends StatefulWidget {
```
Comment (hophongit98) [inline lib/features/add_property/view/widgets/add_rooms_view.dart:9]:
```suggestion
class AddPropertyRoomsView extends StatefulWidget {
```
For consistency with AddPropertyImagesView and AddPropertyFeaturesView

File: lib/features/add_property/view/widgets/add_rooms_view.dart [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+      padding: const EdgeInsets.symmetric(horizontal: 16),
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Expanded(
+            child: Padding(
+              padding: const EdgeInsets.only(top: 24),
+              child: Text(
+                  HatSpaceStrings.of(context).addBedroomViewInstructions,
```
Comment (hophongit98) [inline lib/features/add_property/view/widgets/add_rooms_view.dart:50]:
```suggestion
                  HatSpaceStrings.current.addBedroomViewInstructions,
```

File: lib/features/add_property/view/widgets/add_rooms_view.dart [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+                  style: Theme.of(context).textTheme.displayLarge),
+            ),
+          ),
+          _AddBedroomCounter(
+              counter: bedroomCounter,
+              text: HatSpaceStrings.of(context).bedroomText),
```
Comment (hophongit98) [inline lib/features/add_property/view/widgets/add_rooms_view.dart:56]:
```suggestion
              text: HatSpaceStrings.current.bedroomText),
```

File: lib/features/add_property/view/widgets/add_rooms_view.dart [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+          _AddBedroomCounter(
+              counter: bathRoomCounter,
+              text: HatSpaceStrings.of(context).bathroomText),
```
Comment (hophongit98) [inline lib/features/add_property/view/widgets/add_rooms_view.dart:59]:
```suggestion
              text: HatSpaceStrings.current.bathroomText),
```

**Review by hophongit98 — APPROVED** [review]

**Review by hellomentorplus — APPROVED** [review]

**Pull request merged**